platforms/documentation/docs/src/docs/userguide/releases/upgrading/upgrading_version_8.adoc

```diff
@@ -82,7 +82,9 @@ These methods will be removed in Gradle 9.0.
 
 It is now required to enable stricter validation of plugin types.
 
-This needs to be done in your local build logic (`buildSrc` or included plugins builds). This is also required if you are simply building Gradle plugins.
+The `ValidatePlugins` task is registered by the `java-gradle-plugin`, `groovy-gradle-plugin` and `kotlin-dsl` plugins.
```

Comment:
@eskatos, can you move this entire sub-section from line 80-96:

```
[[validate_plugins_without_stricter_validation]]
==== Not enabling `ValidatePlugins.enableStricterValidation` is deprecated

// Text
```

To a new proper section on line 37:

```
[[changes_9.0]]
== Upgrading from 8.14 and earlier

=== Potential breaking changes

[[validate_plugins_without_stricter_validation]]
==== Not enabling `ValidatePlugins.enableStricterValidation` is deprecated

//Text
```